ticket-booker/src/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
```

Comment:
This needs to be imported to override the default browser stylesheet

Reply:
imported into the index.js file